public/app/features/query/components/QueryEditorRow.tsx

```diff
@@ -594,6 +601,20 @@ function notifyAngularQueryEditorsOfData<TQuery extends DataQuery>(
   setTimeout(editor.digest);
 }
 
+function PluginLinks() {
+  const result = usePluginLinks({ extensionPointId: 'grafana/query-editor-row/actions' });
```

Comment:
would this not need context on what data source the query is for?

Reply:
I plan to remove this before merging and add it later in another PR as it's not clear if this will be a place to put the extension point in. So this code is mostly so it's possible to test it a bit.